pkg/rke/templates/metrics_v0.7.0.go

```diff
@@ -187,7 +198,7 @@ spec:
           periodSeconds: 10
         args:
         - --cert-dir=/tmp
-        - --secure-port=4443
+        - --secure-port=10250
```

Comment:
Kubelet uses `10250` but looks like the change in upstream was intentional `https://github.com/kubernetes/kubernetes/pull/105957#issuecomment-953809831`

Reply:
k8s `v1.28.6` deployed successfully on Rancher with metrics-server `v0.7.0`
![Screenshot from 2024-02-22 01-31-46](https://github.com/rancher/kontainer-driver-metadata/assets/32087475/7d951e80-85b2-4530-84b3-29618163a1f3)